pkg/operator/deploy/deploy.go

```diff
@@ -56,6 +57,7 @@ type Operator interface {
 	Restart(context.Context, []string) error
 	IsRunningDesiredVersion(context.Context) (bool, error)
 	RenewMDSDCertificate(context.Context) error
+	EnsureUpgradeAnnotations(context.Context) error
```

Comment:
```suggestion
	EnsureUpgradeAnnotation(context.Context) error
```

Suggesting this change for clarity's sake since there's only one annotation.